templates/blog/product-cards/_multi_cloud.html

```diff
@@ -0,0 +1,12 @@
+<div class="p-card js-product-card u-hide">
+  <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/04d70172-Multi-Cloud.svg" alt="cloud icon">
+  <hr class="u-sv1">
+  <h3>
+    <a href="/cloud/multi-cloud">What is a multi-cloud?</a>
+  </h3>
+  <p>There is no one size fits all cloud architecture.
+    <br>
+    Developing the optimum cloud strategy requires evaluating your business needs and aligning them with the different solutions available.
```

Comment:
@tytus-kurek any copy change you would like for this card?

Reply:
@caldav I'm fine with having the same content under each product cards, but may I suggest that we title them "Why ..." instead of "What is ...". For example, "Why hybrid cloud?" instead of "What is hybrid cloud?" Such titles would better fit the text below and could drive more engagement. WDYT?

Reply:
This could diminish their SEO value by quite a lot but I do agree that it would be better from a UX perspective. What about changing to Why, then reviewing in a month to see if we got SEO results with it?